keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -349,18 +354,131 @@ feature without providing a configuration, the default configuration will reflec
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [X] Other (treat as last resort)
+  - Details: The cluster administrator SHOULD have access to logs and metrics during cluster upgrade.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A. This feature is short-lived and only active during cluster upgrade.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [X] Metrics
+  - leader_active
+  - other per-controller availability metrics.
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+It would help if every controller that the controller manager hosts expose metrics about their availability.
+However, per-controller metrics are out of scope of this KEP.
+
+### Dependencies
+
+###### Does this feature depend on any specific services running in the cluster?
+
+- API Server
+  - needed for leader election
+    - Impact of its outage on the feature: when leader election timeout, controller managers will lose the leader and exit, causing outage.
+    - Impact of its degraded performance or high-error rates on the feature: delayed or retried operations of leader election.
+
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+Leader Migration uses exactly one more resource of `coordination.k8s.io/v1.Lease` using the standard leader election process.
+Both `kube-controller-manager` and `cloud-controller-manager` will create, update, and watch on the lease.
+
+If the service accounts are not granted access to the lease resources, the RBAC roles of each controller manager may need to modified before the upgrade.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+Type: `controllermanager.config.k8s.io/v1alpha1.LeaderMigrationConfiguration`
+This resource is only for configuration file parsing. The resource should never reach the API server.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+This feature uses exactly one more `coordination.k8s.io/v1.Lease` resource. The RBAC roles of both controller managers will 
+gain additional ~50 bytes because of the new lease under `resourceName`.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+Both `kube-controller-manager` and `cloud-controller-manager` runs another leader election process,
+which may slightly increase CPU usages during leader changes, and it needs extra memory to hold another lease.
```

Comment:
I would phrase these as negligible increases in memory and CPU.

Reply:
Updated.